### PR TITLE
Add permission templates content

### PR DESCRIPTION
**Pull request by rcaballeromx**
Updates the Configure Agentless Scanning procedure and adds the permission templates information.

### CODE REVIEW

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
+== Agentless Permission Templates
+
+Agentless scanning requires specific permissions to work on your self-hosted deployments.
+Different platforms have different permission requirements. 
+To make applying the right permissions easier, you can download permission templates for your specific platforms.
+The following sections explain the different permission templates and what they entail.
+
+=== AWS
+
+You need to download three permission templates as JSON files for AWS cloud account scanning. The templates are needed to enable the different ways accounts are scanned: independent scanning, scan hub account, and target account.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: Can we reword this to say "Upon clicking on "Download" button in AWS configuration for permissions, you will see three files downloaded. "

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+==== Independent Scanning
+
+All scanning happens within the same account.
+This account is independent of other accounts.
+The account requires all  needed permissions to create snapshots, create scanners, etc. 
+The permission template you need to download ends in `_target_user_permissions.json`.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: Please reword to - "To scan accounts in this mode, apply the permission template that ends with: `_target_user_permissions.json` suffix."

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+==== Scan Hub Account
+
+This account is the hub account where you want all account scanning to happen.
+It scans not just the hosts in this account, but it also scans the hosts in other cloud accounts.
+This account requires the permissions needed to run Prisma Cloud Scanners and to access to all other cloud accounts.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: "to access" to just say "access to" all other cloud accounts targeted to scan.

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+These permissions allow the hub account to see and transfer host snapshots from other accounts to this account. 
+The permission template you need to download ends in `_hub_user_permissions.json`.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: Please reword to - "To scan in Hub account setup, apply the permission template that ends with: `_hub_user_permissions.json` to the specific account you would like to use as Hub."

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+==== Target Account
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: please make this a sub heading under the "Scan hub setup"
Essentially there are two ways to scan - Hub setup (1 account scans other accounts). Individual setup (each account scans itself).
In individual setup, there is only file permission template.
In hub setup, there are two: 1 needs to be applied to the hub account, 1 needs to be applied to all target accounts that hub will scan.

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+You need to download three permission templates as JSON files for AWS cloud account scanning. The templates are needed to enable the different ways accounts are scanned: independent scanning, scan hub account, and target account.
+
+==== Independent Scanning
+
+All scanning happens within the same account.
+This account is independent of other accounts.
+The account requires all  needed permissions to create snapshots, create scanners, etc. 
+The permission template you need to download ends in `_target_user_permissions.json`.
+
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc:34]: Can we add a heading equivalent to independent scanning and call it "Scan Hub setup"

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+==== Scan Hub Account
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: lower heading under the above.

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+It scans not just the hosts in this account, but it also scans the hosts in other cloud accounts.
+This account requires the permissions needed to run Prisma Cloud Scanners and to access to all other cloud accounts.
+These permissions allow the hub account to see and transfer host snapshots from other accounts to this account. 
+The permission template you need to download ends in `_hub_user_permissions.json`.
+
+==== Target Account
+
+This is the account that a hub account scans.
+The permissions needed in target accounts are less than those required for independent scanning, but agentless still needs specific permissions to enable the hub account to scan a target account.
+The permission template you need to download ends in `_hub_target_user_permissions.json`.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: "Apply the permission template that ends with:  `_hub_target_user_permissions.json` to the accounts that are being scanned bu Hub account.

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+
+=== Azure
+
+Azure only requires individual account scanning permissions because there is no hub account. Prisma Cloud creates a dedicated PCC resource group, which enables easy cost calculation in the same account scanning. If you don't provide a security group, Prisma Cloud automatically creates one.
+
+=== GCP
+
+You need to download four permission templates as JINJA files for GCP cloud account scanning. 
+Three templates are needed to enable the different ways accounts are scanned: independent scanning, scan hub account, and target account.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc:60]: "Upon clicking on "Download" button in GCP configuration for permissions, you will see four files downloaded."

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+
+
+==== Independent Scanning
+
+All scanning happens within the same account.
+This account is independent of other accounts.
+The account requires all  needed permissions to create snapshots, create scanners, etc. 
+The permission template you need to download ends in `_target_user_permissions.yaml.jinja`.
+
+==== Scan Hub Account
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: same as above, lower heading for this one. A higher heading equivalent to "independent scanning" style to add "Scan Hub setup"

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+This account is the hub account where you want all account scanning to happen.
+It scans not just the hosts in this account, but it also scans the hosts in other cloud accounts.
+This account requires the permissions needed to run Prisma Cloud Scanners and to access to all other cloud accounts.
+These permissions allow the hub account to see and transfer host snapshots from other accounts to this account. 
+The permission template you need to download ends in `_hub_user_permissions.yaml.jinja`.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: "Apply the permission template that ends with: _ `_hub_user_permissions.yaml.jinja` to the account you selected to be a Hub account.

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+==== Target Account
+
+This is the account that a hub account scans.
+The permissions needed in target accounts are less than those required for independent scanning, but agentless still needs specific permissions to enable the hub account to scan a target account.
+The permission template you need to download ends in `_hub_target_user_permissions.yaml.jinja`.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: "Apply the permission template that ends with: `_hub_target_user_permissions.yaml.jinja` to the accounts you selected to be scanned by the Hub account.

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+In GCP, snapshots are created in the hub account. Therefore, the hub account needs additional permissions in the target account. 
+You must apply the following template in the target account to grant the hub account this additional permission.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: "apply an additional template ..

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+The permission template you need to download ends in `_hub_target_access_permissions.yaml.jinja `.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc]: ..template to be applied ends in ..

**Review by Aqsaxvv092 — COMMENTED** [review]
Please review

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 15]
```diff
@@ -0,0 +1,68 @@
+== Agentless Permission Templates
+
+Agentless scanning requires specific permissions to work on your self-hosted deployments.
+Different platforms have different permission requirements. 
+To make applying the right permissions easier, you can download permission templates for your specific platforms.
```
Comment (Aqsaxvv092) [inline compute/admin_guide/configure/agentless-permission-templates.adoc:5]: Can we add the below please here in intro -- 
"There are two ways you can set up agentless scanning with Prisma Cloud.
Scan all hosts of a cloud account within the same cloud account - Independent Scanning. In this mode, the scanners are spin up in every account that is scanned. 
or
Scan all hosts of a cloud account, called target account(s), in a dedicated cloud account, called hub account. - Scan hub setup. In this mode, the scanners are spin up only in the hub account and not the target accounts being scanned by hub.

File: compute/admin_guide/configure/agentless-permission-templates.adoc (hunk 5 above)
Comment (rcaballeromx) [reply compute/admin_guide/configure/agentless-permission-templates.adoc]: I've added the subsections and some context to the document. PTAL @Aqsaxvv092

File: compute/admin_guide/configure/agentless-permission-templates.adoc (hunk 6 above)
Comment (rcaballeromx) [reply compute/admin_guide/configure/agentless-permission-templates.adoc:34]: I've added the subheadings and context.

**Review by Aqsaxvv092 — APPROVED** [review]

File: compute/admin_guide/configure/agentless-permission-templates.adoc [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+The following sections explain the different permission templates and what they entail.
+
+=== AWS
+
+You need to download three permission templates as JSON files for AWS cloud account scanning. The templates are needed to enable the different ways accounts are scanned: independent scanning, scan hub account, and target account.
+
+==== Independent Scanning
```
Comment (Pubs-MV) [inline compute/admin_guide/configure/agentless-permission-templates.adoc:26]: @rcaballeromx @Aqsaxvv092 I understand that "Independent" means that we include the permissions for hub and target together and it is for scanning within each cloud account.

Is this related to the "individual account" and "multiple account" phrasing in the configure agentless scanning topic? If not, how does it map for our users?
